opentelemetry-api/src/logs/record.rs

```diff
@@ -79,6 +80,38 @@ pub enum AnyValue {
     Map(OrderMap<Key, AnyValue>),
 }
 
+impl fmt::Display for AnyValue {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            AnyValue::Int(v) => v.fmt(fmt),
+            AnyValue::Double(v) => v.fmt(fmt),
+            AnyValue::String(v) => fmt.write_str(v.as_str()),
+            AnyValue::Boolean(v) => v.fmt(fmt),
+            AnyValue::Bytes(v) => common::display_array_str(v.as_slice(), fmt),
```

Comment:
Not sure if `display_array_str` is write method to format byte vector, as it will try to display as string. We can have a similar method `display_array_byte` which print the bytes as a sequence of hexadecimal values:

`write!(f, "{:02x}", t)?;`